### PR TITLE
Map MS partof to RO

### PR DESCRIPTION
**Pull request by matentzn**
Fixes https://github.com/EBISPOT/ols4/issues/600

Ideally, in a future PR, you should consider to import RO rather than copy pasting the semantics here (i.e. transitivity).

### CODE REVIEW

**Review by ypriverol — APPROVED** [review]

File: psi-ms.obo [hunk 1]
```diff
@@ -45,6 +45,7 @@ name: has_units
 [Typedef]
 id: part_of
 name: part_of
+xref: BFO:0000050
```
Comment (ypriverol) [inline psi-ms.obo:48]: @matentzn Can you update the version of the ontology. See the failing Actions https://github.com/HUPO-PSI/psi-ms-CV/actions/runs/7638645173/job/20810093289

Comment (matentzn) [reply psi-ms.obo:48]: Feel free to close this PR and redo it, I just wanted to show you how you can map your part of to RO, so it displays correctly in OLS and interoperates with other ontologies. You can also directly edit my PR and make the required changes!

Comment (ypriverol) [reply psi-ms.obo:48]: Done. Thanks a lot for the tip.

**Review by edeutsch — APPROVED** [review]
It would be funny if after all this pain, the solution is as simple as this..

**Comment by ypriverol** [comment]
I merged the PR to see the changes and opened the owl created in Protege as suggested; and it looks for me that the problem still remains 

Screenshot 2024-01-24 at 20 17 51

Any advice @matentzn ?

**Comment by matentzn** [comment]
This is a weakness in Protege I think. If you look at the peptide classes, you can see peptide attribute underneath, with the blue arrow..

image

Lets see how Protege will display this..

**Comment by matentzn** [comment]
(I think the problem in Protege is that it will include the is a hierarchy no matter what, and superimposes the other "relation ships" after the fact - which means that what you see in terms of top level roots are all the roots according to the is a hierarchy)

**Pull request merged**